mechanicalsoup/form.py

```diff
@@ -306,9 +306,11 @@ def new_control(self, type, name, value, **kwargs):
     def choose_submit(self, submit):
         """Selects the input (or button) element to use for form submission.
 
-        :param submit: The bs4.element.Tag (or just its *name*-attribute) that
-            identifies the submit element to use. If ``None``, will choose the
-            first valid submit element in the form, if one exists.
+        :param submit: The :class:`bs4.element.Tag` (or just its
+            *name*-attribute) that identifies the submit element to use. If
+            ``None``, will choose the first valid submit element in the form,
+            if one exists. If ``False``, will not use any submit element;
+            this is useful for simulating AJAX requests, for example.
```

Comment:
Would you mind extending the documentation of the `False` case to the `btnName` argument of `stateful_browser.submit_selected`?

Reply:
Sure. I had actually left it in initially, then removed it for economy's sake, but it's probably helpful to have it in there, indeed.